members/PulsatingGenius.md

```diff
@@ -0,0 +1,4 @@
+---
+name: Aman Anand
+github_handle: PulsatingGenius
+---
```

Comment:
This looks really good but this was supposed to be implemented in a different format, you can see how others have done this [here](https://github.com/SST-OSS/club-onboarding/tree/main/members).

Reply:
updated as told